controller/status.go

```diff
@@ -213,6 +213,13 @@ var (
 	StatusIndex = []StatusContext{
 		{
 			FleetCurrent:  "inactive",
+			FleetDesired:  "launched",
+			SystemdActive: "active",
+			SystemdSub:    "running",
+			Aggregated:    StatusRunning,
+		},
```

Comment:
this = the line in the URL. This is just broken because that line doesn't contain inactive. As we more or less want to ignore the fleet state anyway, we could put a `*` there.